### ISSUE TITLE
Use `Sonner` instead of `@yamada-ui/motion` in `Notice` component

### ISSUE THREAD

**Issue by koralle** [yamada-ui/yamada-ui#3425]
### Description

The Notice component uses `@yamada-ui/motion` to add interactions.
We would like to replace this with [Sonner](https://sonner.emilkowal.ski/).

### Problem Statement/Justification

`@yamada-ui/motion` uses [`Framer Motion`](https://www.framer.com/motion/) internally, but it is a little short of what we want to do with Yamada UI.

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

**Comment by sarthakvijayvargiya** [comment]
Hi @koralle  can i work on this

**Comment by koralle** [comment]
@sarthakvijayvargiya 
That's fine.

**Comment by koralle** [comment]
@sarthakvijayvargiya 
Yes, please! The issue is all yours. Thank you for wanting to help! 🙌

**Comment by hirotomoyamada** [comment]
@sarthakvijayvargiya

Hi, Thanks for the working on this issue!

We will return this issue assignment to initial state.
If you are working on this issue and would like to be reassigned, please contact us.
